### PR TITLE
Support Rails 6

### PR DESCRIPTION
**Pull request by indirect**
Bumps the version to 5.0, drops support for Ruby 2 and Rails 5, and fixes things up so that it works with Rails 6.

### CODE REVIEW

**Comment by indirect** [comment]
Oh it also switches from Travis to Github Actions for CI.

File: README.rdoc [hunk 1]
```diff
@@ -1,6 +1,4 @@
-= Rails Footnotes
-
-{<img src="https://travis-ci.org/Intrepidd/rails-footnotes.svg?branch=master" />}[https://travis-ci.org/Intrepidd/rails-footnotes]
+= Rails 6 Footnotes
```
Comment (Intrepidd) [inline README.rdoc:1]:
```suggestion
= Rails Footnotes
```

File: rails-footnotes.gemspec [hunk 2]
```diff
@@ -6,23 +6,16 @@ Gem::Specification.new do |s|
   s.name        = "rails-footnotes"
   s.version     = Footnotes::VERSION
   s.platform    = Gem::Platform::RUBY
-  s.authors     = ["Roman V. Babenko", "José Valim", "Keenan Brock", "Duane Johnson", "Adrien Siami"]
-  s.email       = ["romanvbabenko@gmail.com"]
-  s.homepage    = "http://github.com/josevalim/rails-footnotes"
+  s.authors     = ["Roman V. Babenko", "José Valim", "Keenan Brock", "Duane Johnson", "Adrien Siami", "André Arko"]
+  s.email       = ["andre@arko.net"]
+  s.homepage    = "http://github.com/indirect/rails-footnotes"
```
Comment (Intrepidd) [inline rails-footnotes.gemspec:11]: I can work out the transfer of the repository under your username, would you want it?

File: rails-footnotes.gemspec [hunk 3]
```diff
@@ -6,23 +6,16 @@ Gem::Specification.new do |s|
...
 
-  s.rubyforge_project = "rails-footnotes"
-
-  s.add_dependency "rails", ">= 3.2"
-
-  s.add_development_dependency "rake", '< 11.0'
-  s.add_development_dependency "rspec-rails", '~> 3.3.2'
-  s.add_development_dependency "sprockets-rails", '~> 2.0'
-  s.add_development_dependency "capybara"
+  s.add_dependency "rails", "~> 6.0"
```
Comment (Intrepidd) [inline rails-footnotes.gemspec:15]: 🎉

**Review by Intrepidd — APPROVED** [review]
Thank you so much :) let's figure out the transition details

File: rails-footnotes.gemspec (hunk 2 above)
Comment (indirect) [reply rails-footnotes.gemspec:11]: Sure, that works for me. Maybe we merge this PR, and then transfer it after that?

**Comment by Intrepidd** [comment]
Thanks, I can't transfer it as it already exists (your fork) 

Do you mind deleting or renaming it so I can proceed with the transfer?

**Comment by indirect** [comment]
@Intrepidd done! sorry about that, should have realized lol.

**Comment by Intrepidd** [comment]
Just sent you the request, gonna see about rubygems now

**Comment by indirect** [comment]
Awesome, accepted! I think it’s just `gem owner rails-footnotes -a andre@arko.net` 👍🏻

**Pull request merged**